index.js

```diff
@@ -0,0 +1,80 @@
+var run = require('comandante')
+var _ = require('lodash')
+var Q = require('kew')
+var ipfs = require('ipfs-api')
+
+function configureNode (node, conf, cb) {
+  var delay = 0
+  var promises = _.map(conf, function (value, key) {
+    var def = Q.defer()
+    setTimeout(function () {
+      run('ipfs', ['config', key, value])
+        .on('error', function (err) { cb(err) })
+        .on('end', function () { def.resolve(true) })
+    }, delay * 200)
+    delay++
```

Comment:
It seems there can be a race condition when you're setting two config variables at the same time. I will make an issue of this in my next sitting.